test/Microsoft.ML.Benchmarks/Numeric/Ranking.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using System.IO;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class Ranking
+    {
+        public string _mslrWeb10k_Validate;
+        public string _mslrWeb10k_Train;
+        public string _mslrWeb10k_Test;
+        private string _modelPath_MSLR;
+
+        [GlobalSetup(Targets = new string[] {
+            nameof(TrainTest_Multiclass_MSLRWeb10K_Ranking_FastTree),
+            nameof(TrainTest_Multiclass_MSLRWeb10K_Ranking_LightGBM) })]
+        public void SetupTrainingSpeedTests()
+        {
+            _mslrWeb10k_Validate = Path.GetFullPath(TestDatasets.MSLRWeb.validFilename);
+            _mslrWeb10k_Train = Path.GetFullPath(TestDatasets.MSLRWeb.trainFilename);
+            
+            if (!File.Exists(_mslrWeb10k_Validate))
+            {
+                throw new FileNotFoundException($"Could not find {_mslrWeb10k_Validate} Please ensure you have run 'build.cmd -- /t:DownloadExternalTestFiles /p:IncludeBenchmarkData=true' from the root");
```

Comment:
Let's put this format string in a const somewhere, so it doesn't need to be duplicated in multiple places.